sdk/include/reactos/libs/libxml/xmlversion.h

```diff
@@ -475,6 +484,15 @@ XMLPUBFUN void XMLCALL xmlCheckVersion(int version);
  * Macro used to indicate to GCC the parameter are printf like
  */
 #define LIBXML_ATTR_FORMAT(fmt,args)
+/**
+ * XML_DEPRECATED:
+ *
+ * Macro used to indicate that a function, variable, type or struct member
+ * is deprecated.
+ */
+#ifndef XML_DEPRECATED
+#define XML_DEPRECATED
```

Comment:
Not the purpose of a sync, but would it make sense to add `__declspec(deprecated)` or `[[deprecated]]` here?

Reply:
Yes, it would. The libxml2 2.10.0 sync actually breaks libxslt because it deprecates a bunch of stuff (libxslt 1.1.36 fixes that) -- but only on GCC. So it would indeed be nice to give MSVC a hint too.